docker/consumedir.dockerfile

```diff
@@ -1,19 +1,13 @@
 ## CONSUMEDIR
 
-ARG VERSION=
-ARG REPO=
-
-# hack to use args in from
-FROM ${REPO}:base-binaries-${VERSION} as docspell-base-binaries
-
-
-FROM ${REPO}:base-${VERSION}
+FROM alpine:latest
+LABEL maintainer="eikek0 <eike@docspell.org>"
 
 RUN apk add --no-cache curl bash inotify-tools
 
-COPY --from=docspell-base-binaries /opt/docspell-tools /opt/docspell-tools
+COPY ./tools/consumedir /opt/consumedir
```

Comment:
I'm sorry for a bunch of stupid questions (I'm not docker versed): This seems now to copy the script from the current directory? Do you think the previous `--from=…` is not needed anymore?

Reply:
Yes, exactly. The previous source (`--from=docspell-base-binaries`) includes all the sources via a similar COPY copy statement and compiles them afterwards. Since we're only interested in the bash scripts, that don't need any compilation, I can just get them from the current directory.